src/components/Verification/VerificationButton.jsx

```diff
@@ -0,0 +1,281 @@
+                  gap: '10px'
+                }}
+              >
+                <Button
+                  onClick={closeModal}
+                  style={{
+                    flex: 1,
+                    backgroundColor: 'gray',
+                    color: 'white',
+                    borderRadius: '4px',
+                    padding: '4px'
+                  }}
+                >
+                  Cancel
+                </Button>
+                <Button
+                  onClick={() => {
+                    if (btoa(password) == PASSWORD) {
+                      setIsAdmin(true);
+                    } else {
+                      setLoginError('Invalid password');
+                    }
+                  }}
+                  style={{
+                    flex: 1,
+                    backgroundColor: 'blue',
+                    color: 'white',
+                    borderRadius: '4px',
+                    padding: '4px'
+                  }}
+                >
+                  Login
+                </Button>
+              </div>
+            </div>
+          )}
+          {isAdmin && (
+            <div>
+              {!hasBeenUpdated && (
+                <div>
+                  {/* As an admin, you can either mark this as verified, unverified, or inactive */}
+                  <p>
+                    Please select the change you would like to make. This was
+                    previously marked as{' '}
+                    {resource.verification.verified ? 'VERIFIED' : 'UNVERIFIED'}{' '}
+                    by {resource.verification.last_modifier}.
+                  </p>
+
+                  <Button
+                    onClick={markAsVerified}
+                    style={{
+                      backgroundColor: 'green',
+                      color: 'white',
+                      padding: '10px',
+                      margin: '5px',
+                      borderRadius: '5px',
+                      width: '100%'
+                    }}
+                  >
+                    MARK AS VERIFIED
+                  </Button>
+                  <Button
+                    onClick={markAsUnverified}
+                    style={{
+                      backgroundColor: 'orange',
+                      color: 'white',
+                      padding: '10px',
+                      margin: '5px',
+                      borderRadius: '5px',
+                      width: '100%'
+                    }}
+                  >
+                    MARK AS UNVERIFIED
+                  </Button>
+                  <Button
+                    onClick={markAsInactive}
+                    style={{
+                      backgroundColor: 'red',
+                      color: 'white',
+                      padding: '10px',
+                      margin: '5px',
+                      borderRadius: '5px',
+                      width: '100%'
+                    }}
+                  >
+                    MARK AS INACTIVE
```

Comment:
nit: MUI buttons by default are uppercase - but I'm not sure if we overridden this in the theme